### DISCUSSION TITLE
Chat Color - Add Modifiers

### DISCUSSION THREAD

**Discussion by Harvle** [Zrips/CMI#8220] [Feature Requests]
Within the same menu that comes up with using `/chatcolor`, would it be possible to add a row at the bottom of the menu for "modifiers", like bold, italic, strikethrough, etc.

These buttons would be a toggle, and would add to the end of the selected color. Just like the colors, they'd use the `cmi.colors.publicmessage.[bold|italic|magic|reset|strikethrough|underline]`.

Examples of ways the `%cmi_user_chatcolor%` placeholder would end up being after this feature:
- White with italic: `&f&o`
- Gold with bold and underline: `&6&l&n`
- Custom hex "Brown" with magic: `{#brown}&k`

**Comment by CloudeLecaw** [comment]
they where added. so request can be closed :-)
